src/lab2.py

```diff
@@ -65,55 +65,67 @@ def lex(body):
         breakpoint("lex", text)
     return text
 
-WIDTH, HEIGHT = 800, 600
```

Comment:
I got rid of these global variables, since they were creating a lot of headaches when trying to set them
from different modules, and upgrade the compiler to match. It seems just as good to pass them as
constructor arguments to Browser.